atom/browser/api/atom_api_window.cc

```diff
@@ -914,6 +915,65 @@ void Window::ResetBrowserView() {
   browser_view_.Reset();
 }
 
+void Window::AddBrowserView(v8::Local<v8::Value> value) {
+  mate::Handle<BrowserView> browser_view;
+  if (value->IsNull() || value->IsUndefined()) {
```

Comment:
There is no need to check `null`, type check can be done with:

```c++
if (mate::ConvertFromV8(isolate(), value, &browser_view)) {
} else {
  // Throw type error.
}
```

`Window::RemoveBrowserView` can also be modified in the same way.

Reply:
Work of ConvertFromV8 is done by template implementation for corresponding type. For most types this indeed have check for null. But mate::Handle<> FromV8 converter does something opposite. 
```
 if (val->IsNull() || val->IsUndefined()) {
       *out = mate::Handle<T>();
       return true;
     }
```
And that lead to crash if null is passed. So I have to add check back. 
But change it to value->IsObject() 
Is it OK?

Do I have to add "Throw type error"? Prev API does not do it and as AddBrowserView works now as fall back for SetBrowserView. Then SetBrowserView start to throwing error too. But sending null to setBrowserView is required way to remove BrowserView what was set before.